app/screens/BookDetail/layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import BookDetailOptions from './components/BookDetailOptions';
+import { styles } from './styles';
+
+function BookDetail(props) {
+  return (
+    <View style={styles.container}>
+      <BookDetailOptions {...props} />
+    </View>
+  );
+}
+
+export default BookDetail;
```

Comment:
I think this `layout.js` is not really necessary. You can call `BookDetailOptions` directly in the `index`

Reply:
Yes, in this case its not neccesary!. But, i need keep it because in the next PR i added more components on `layout.js`.